server/src/server.ts

```diff
@@ -342,6 +342,16 @@ function addClassPropertiesMethodsParentClassesAndTraits(toReturn: CompletionIte
     }
 }
 
+function stringifyMethodParams(subNode): string
+{
+    var params = [];
+    subNode.params.forEach(param => {
+        if(param.optional){return;}
```

Comment:
Optional parameters should probably still be shown, but with their default value -
eg `($required, $optional = "test")`.

I _think_ this value is available under `param.value`, but you'll have to check